tracer/src/Datadog.Trace/Debugger/LiveDebugger.cs

```diff
@@ -88,11 +88,11 @@ internal static LiveDebugger Create(ImmutableDebuggerSettings settings, IDiscove
 
         private void Initialize()
         {
-            if (!_settings.Enabled)
-            {
-                Log.Information("Live Debugger is disabled. To enable it, please set DD_DEBUGGER_ENABLED environment variable to 'true'.");
-                return;
-            }
+            // if (!_settings.Enabled)
+            // {
+            //     Log.Information("Live Debugger is disabled. To enable it, please set DD_DEBUGGER_ENABLED environment variable to 'true'.");
+            //     return;
+            // }
```

Comment:
I'm not sure how this change contributes to disabling the Live Debugger... (?)

Reply:
🤦 Not enough coffee this morning. Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/3012/commits/f893fa2324c1776a56b31fd7c00214b20d73db96.

Reply:
Slightly concerning that neither me nor @pierotibou spotted this either 🤦‍♂️ 😬